Student/Student/Routes.swift

```diff
@@ -68,7 +68,10 @@ let routeMap: [String: RouteHandler.ViewFactory?] = [
 
     "/:context/:contextID/announcements/:announcementID": nil,
 
-    "/courses/:courseID/assignments": nil,
+    "/courses/:courseID/assignments": ExperimentalFeature.assignmentListGraphQL.isEnabled ? { url, params in
+        guard let courseID = params["courseID"] else { return nil }
+        return AssignmentListViewController.create(courseID: courseID)
+        } : nil,
```

Comment:
nitpick: outdent

Reply:
😬 i'm anti fighting against the IDE...hesitant to do that.